sdk/src/server-api/sc/networking/clients/LobbyClient.java

```diff
@@ -126,7 +103,7 @@ private void onGameLeft(String roomId) {
   }
 
   private void onGameJoined(String roomId) {
-    for (ILobbyClientListener listener : this.listeners) {
+    for (ILobbyClientListener listener : new ArrayList<>(this.listeners)) {
```

Comment:
Just curious: why?

Reply:
ConcurrentModificationExceptions - I didn't bother to debug them because that code will soon be gone anyways